### PR TITLE
added alt for httlads

### PR DESCRIPTION
**Pull request by mccoyd15**
Added alt text on images from ch 1-6.

### CODE REVIEW

File: _sources/WorldFacts/cs2_data_analysis.rst [hunk 1]
```diff
@@ -343,7 +343,7 @@ Let's write all of the above calls into a single line.
 
 
 .. image:: Figures/visualization_2.png
-
+:alt: This is a histogram depicting protecting minority score.
```
Comment (bnmnetp) [inline _sources/WorldFacts/cs2_data_analysis.rst:346]: Thanks for doing this.  The :alt: options need to be indented at least to the I in image or more commonly we indent them by 4 spaces.

**Review by bnmnetp — CHANGES_REQUESTED** [review]
Please fix the indentation of the `:alt:` options.

Thanks!

File: _sources/WorldFacts/cs2_data_analysis.rst (hunk 1 above)
Comment (ConceptaNjolima) [reply _sources/WorldFacts/cs2_data_analysis.rst:346]: Thank you. We will fix the indentation

File: _sources/WorldFacts/cs2_graphing_business_data.rst [hunk 2]
```diff
@@ -450,12 +450,14 @@ above and apply that to the world below.
 
 
 .. image:: Figures/WorldFactbook_74_0.png
+:alt: This is the world map without color.
```
Comment (bnmnetp) [inline _sources/WorldFacts/cs2_graphing_business_data.rst:453]: Indentation still needs fixing

Comment (bnmnetp) [reply _sources/WorldFacts/cs2_graphing_business_data.rst:453]: BTW, building the book is a good way to catch all of these, as the bad indentations will cause errors.  Thanks.

**Comment by mccoyd15** [comment]
Hello Brad, we are still in the process of building the book. So, we are not able to see the mistakes yet. We will submit a new pull request when those changes are finalized.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: _sources/WorldFacts/cs2_graphing_business_data.rst (hunk 2 above)
Comment (ConceptaNjolima) [reply _sources/WorldFacts/cs2_graphing_business_data.rst:453]: We are working on building the book with the fixed indentation before we send in a new pull request. We are sorry that this is taking long.

**Comment by mccoyd15** [comment]
Had a merge conflict. We will close this pull request and submit a new pull request with the merge conflict resolved

**Pull request closed without merging**